packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -479,6 +485,256 @@ export async function getTransactionCount<ReturnFormat extends DataFormat>(
 
 	return format({ format: 'uint' }, response as Numbers, returnFormat);
 }
+class SendTxHelper<
```

Comment:
as its another class so place it in eth [utils dir ](https://github.com/web3/web3.js/tree/4.x/packages/web3-eth/src/utils)

Reply:
it is used only in `web3-eth` package. I will move it to another file in `web3-eth`